src/components/InterviewList/InterviewList.tsx

```diff
@@ -202,96 +229,178 @@ export class InterviewList extends React.Component<InterviewListProps, Interview
     render() {
         const roles = (store.getState().managementState.auth.currentUser.roles);
         const isAdmin = (roles.includes('admin') || roles.includes('staging-manager') || roles.includes('trainer'));
+        // convert interview array to place array
+        const arrAssociateEmail1 = this.props.listOfInterviews.map((item) => item.associateEmail);
```

Comment:
If these variables aren't needed, then they shouldn't be declared when we could avoid it by chaining these operations.

```
const result = this.props.listOfInterviews.map( ...)
                   .map(...)
                   .filter(...)
                   ...
````